uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 sensor_temperature              # Kelvin
+float16 static_air_temperature          # Kelvin
```

Comment:
what comment do you advise to be put here? will this do : `this field should contain the RAW temperature reading from the sensor needed for actual static_air_temperature calculation`